core/domain/exp_jobs_one_off.py

```diff
@@ -328,3 +328,25 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+class FallbackOneOffJob(jobs.BaseMapReduceJobManager):
+    """Job that outputs a list of fallbacks."""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_services.get_exploration_from_model(item)
+        for state_name, state in exploration.states.iteritems():
+            for fallback in state.interaction.fallbacks:
+                yield (state_name.encode('utf-8'),
```

Comment:
This is definitely the right idea, but don't you want a bit more info than this? Presumably the concatenation "exp_id.state_name" would be the key, and wouldn't you want to get some sense of the order of the fallbacks -- so maybe key by "exp_id.state_name.num_answer_attempts"?